subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleVersionsPluginSmokeTest.groovy

```diff
@@ -16,8 +16,11 @@
 
 package org.gradle.smoketests
 
+import spock.lang.Ignore
+
 import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 
+@Ignore
```

Comment:
can you add a reason to the ignore? e.g., `@Ignore("this causes a deadlock on Gradle 5.x")`

Reply:
Done.